### PR TITLE
solidity-coverage version update

### PR DESCRIPTION
**Pull request by kautukkundan**
**What does this pull request do? Explain your changes. (required)**

updated solidity-coverage version which supports the london upgrade mentioned in #457 

**Specific updates (required)**

- updated solidity-coverage version to 0.7.17
- removed setting initial base fee to 0 

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Checklist:**

- [ ] README and other documentation updated
- [ ] All tests using `yarn test` pass

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -600,12 +600,7 @@
     "@nodelib/fs.scandir" "2.1.5"
     fastq "^1.6.0"
 
-<<<<<<< HEAD
 "@nomiclabs/hardhat-ethers@npm:hardhat-deploy-ethers", hardhat-deploy-ethers@^0.3.0-beta.10:
-  name "@nomiclabs/hardhat-ethers"
-=======
-"@nomiclabs/hardhat-ethers@npm:hardhat-deploy-ethers@^0.3.0-beta.10", hardhat-deploy-ethers@^0.3.0-beta.10:
->>>>>>> 90d5f5e... libs/utils: rework utilities and update compiler version and openzeppelin version
```
Comment (yondonfu) [inline yarn.lock]: I'm surprised that this conflict annotation didn't break CI before.

**Review by yondonfu — CHANGES_REQUESTED** [review]
Looks like tests are [failing in CI](https://github.com/livepeer/protocol/pull/472/checks?check_run_id=3495374200) due to a missing property on a type.

File: yarn.lock (hunk 1 above)
Comment (kautukkundan) [reply yarn.lock]: Actually the CI is failing because the BondingManager contract was updated but the corresponding test cases were not.

Comment (yondonfu) [reply yarn.lock]: Yep noticed that. Was just noting here that I was surprised that this particular conflict annotation didn't cause problems on its own. Would've expected it to mess up the yarn install 🤷

Comment (kautukkundan) [reply yarn.lock]: yeah, this is surprising to me too

I was trying to reply to the previous comment `missing property on a type`. Somehow ended up replying in this thread 😅

**Comment by kautukkundan** [comment]
rebased from next after merging the CI fix

**Review by yondonfu — APPROVED** [review]
LGTM :shipit:

**Pull request merged**